access-front-door/run.sh

```diff
@@ -0,0 +1,10 @@
+# bin/bash
+if [[ -z "$1" ]]; then
+    echo "usage: $0 [device_id]"
+    mpremote devs
+    exit 1;
+fi
+
+mpremote connect id:$1 reset
+mpremote connect id:$1 cp -r src/* :
+mpremote connect id:$1 run ./src/main.py
```

Comment:
This is awesome for debugging -- but because it puts the files under `src` they're not run whenever the chip is not being called via `mpremote`, right?
Do we need a different script to put the `main.py` on the root level? Maybe we should have it use `esptool.py` also to make sure the chip has a completely fresh install?

Reply:
Does it put it under /src/ on the esp32? I copied this code from a pico project and it put it in root.

`run ./src/main.py` doesn't cause the devices main.py to run but the file relative to your current folder